src/main/markdown/eco/add-lib.md

```diff
@@ -0,0 +1,26 @@
+# Update Information
+
+We will add or update information on request. 
+
+## Request for adding Libraries to the Eco System site
+
+The GWT eco system is a living system. New libs will be created, other will be end of life. Also we know, that there are
+much more libraries outside. If you think, one library not listed here is worth to be mentioned, [open an issue
+on our tracker](https://github.com/gwtproject/gwt-site/issues/new/choose). This will trigger a process to get mentioned in the eco system content.
+
+We will expect:
+
+* The Issue must be created by a contributor of the library
+* The Library must be under active development
+* The Library must be well documented
+* The Library must be deployed to Maven central
+* The Library must be an Open Source project
+* The Library requires a Apache 2 Licence
```

Comment:
I don't think we need both bullet points - if it must be Apache v2, we don't need the open source license.

With that said, why should Apache v2 be a requirement (closure-compiler has a MPL dependency, for example)? And while it has become less popular to use closed source UI libraries on the web, it seems reasonable that as long as it is labeled accordingly that it be allowed?

Reply:
Yes, you are right. I added this because I want to avoid adding paid libraries. I don't want to promote vommercial libraries. So I remove the **The Library requires a Apache 2 License**.

Done.